### PR TITLE
Add new, empty eviction free site.

### PR DESCRIPTION
**Pull request by toolness**
This adds a brand new site for us to build EFNYC 2.0 (NYS Declaration Letter) off of.

Currently the site is extremely barebones, to be fleshed out later:

> image

The site can be visited by changing a Site object to have "evictionfree" in its name in the Django admin (this is similar to how we do things for norent).

### CODE REVIEW

File: frontend/lib/analytics/amplitude.ts [hunk 1]
```diff
@@ -233,5 +233,8 @@ export function getAmplitudePageType(pathname: string): string {
       return "JustFix " + getJustfixPageType(pathname);
     case "NORENT":
       return "NoRent " + getNorentPageType(pathname);
+    case "EVICTIONFREE":
+      // TODO: Return the type of page it is for EvictionFree.
+      return "EvictionFree";
```
Comment (toolness) [inline frontend/lib/analytics/amplitude.ts:238]: We should file a github or clubhouse issue for this to make sure we don't forget about it.

File: frontend/lib/pages/cross-site-terms-opt-in.tsx [hunk 2]
```diff
@@ -28,6 +28,10 @@ export function hasLoggedInUserAgreedToTerms(s: AllSessionInfo): boolean {
 
     case "NORENT":
       return s.onboardingInfo.agreedToNorentTerms;
+
+    case "EVICTIONFREE":
+      // TODO: Need to figure out if we need separate terms for EvictionFree!
+      return s.onboardingInfo.agreedToJustfixTerms;
```
Comment (toolness) [inline frontend/lib/pages/cross-site-terms-opt-in.tsx:34]: We should file a github or clubhouse issue for this to make sure we don't forget about it.

File: frontend/lib/ui/page.tsx [hunk 3]
```diff
@@ -41,6 +41,9 @@ function getSiteBaseName(siteType: SiteChoice): string {
 
     case "NORENT":
       return "NoRent.org";
+
+    case "EVICTIONFREE":
+      return "EvictionFree.org";
```
Comment (toolness) [inline frontend/lib/ui/page.tsx:46]: We may need to change this based on what our final domain name is, so we should file an issue to address this later.

File: frontend/lib/ui/privacy-info-modal.tsx [hunk 4]
```diff
@@ -18,6 +18,10 @@ function getURLforSite(baseURL: string, site: SiteChoice): string {
 
     case "NORENT":
       return `${baseURL}-norent`;
+
+    case "EVICTIONFREE":
+      // TODO: Figure out if we need a separate privacy policy / terms for EvictionFree!
+      return baseURL;
```
Comment (toolness) [inline frontend/lib/ui/privacy-info-modal.tsx:24]: Again, file an issue to address this later.

File: frontend/lib/evictionfree/route-info.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+export const EvictionFreeRoutes = createRoutesForSite(
+  createLocalizedRouteInfo,
+  {
+    /**
+     * Example pages used in integration tests, and other
+     * development-related pages.
+     */
+    dev: createDevRouteInfo("/dev"),
+  }
+);
```
Comment (sraby) [inline frontend/lib/evictionfree/route-info.ts:23]: Nice! love the new `route-info.ts` name, makes it v clear

**Review by sraby — APPROVED** [review]
Looks great!

**Pull request merged**